setup.cfg

```diff
@@ -171,6 +171,7 @@ filterwarnings =
     # an astropy table inside a test, which is generally only done when
     # debugging.
     ignore:the imp module is deprecated in favour of importlib:DeprecationWarning:ipykernel.iostream
+    ignore:The `use_readline` parameter:DeprecationWarning:
```

Comment:
Where does this come from?

Reply:
Accidentally committed change I need for PyCharm and pytest and ipthon shell to work. I've removed it